src/egon/data/datasets/etrago_setup.py

```diff
@@ -129,8 +129,35 @@ def __init__(self, dependencies):
 
 
 class EgonPfHvBus(Base):
+
+    source_list = [
+        sources()["egon-data"],
```

Comment:
Can we really cite this eGon-data as a source in our metadata?

Reply:
We added it like this to every table, so I think it's fine.